cpp/src/io/csv/csv_reader.cu

```diff
@@ -753,7 +753,8 @@ gdf_error read_csv(csv_read_arg *args)
 		for ( int x = 0; x < raw_csv->num_actual_cols; x++) {
 
 			std::string temp_type 	= args->dtype[x];
-			gdf_dtype col_dtype;
+                        // Initialize col_dtype as GDF_invalid
```

Comment:
This comment literally only says what the next code line is doing, so it doesn't add anything. Can you remove?

Reply:
Sure. I will add more description in my next commit. Thanks for reviewing.

Reply:
I have removed description added in intermediate commit- because `convertStringToDtype` is already setting `GDF_invalid` in case non-supported data type. This change will only initialize `col_dtype` . Thanks!

Reply:
Great. Can you resolve this conversation if you are done, thanks.